ActionButtonItem.js

```diff
@@ -85,28 +85,27 @@ export default class ActionButtonItem extends Component {
     };
 
     if (position !== "center")
-      buttonStyle[position] = (this.props.parentSize - size) / 2;
+      animatedViewStyle[position] = (this.props.parentSize - size) / 2;
 
     const Touchable = getTouchableComponent(this.props.useNativeFeedback);
 
-    const parentStyle = isAndroid &&
-      this.props.fixNativeFeedbackRadius
-      ? {
-          height: size,
-          marginBottom: spacing,
-          right: this.props.offsetX,
-          borderRadius: this.props.size / 2
+    const parentStyle = {
+          marginHorizontal: this.props.offsetX,
+          marginBottom: spacing + SHADOW_SPACE,
+          borderRadius: this.props.size / 2,
         }
-      : {
-          paddingHorizontal: this.props.offsetX,
-          height: size + SHADOW_SPACE + spacing
-        };
+
     return (
       <Animated.View
         pointerEvents="box-none"
-        style={[animatedViewStyle, parentStyle]}
+        style={[animatedViewStyle]}
       >
```

Comment:
Android needs the shadow set on the outer view so that the shadows dont get cropped